android/AndroidManifest.xml

```diff
@@ -31,7 +31,6 @@ file, You can obtain one at http://mozilla.org/MPL/2.0/.
         <activity
             android:configChanges="orientation|uiMode|screenLayout|screenSize|smallestScreenSize|layoutDirection|locale|fontScale|keyboard|keyboardHidden|navigation|mcc|mnc|density"
             android:name="org.mozilla.firefox.vpn.qt.VPNActivity"
-            android:process=":QtOnlyProcess"
```

Comment:
Note: glean expects to live on the main process - which is fine, we just don't want qt and WireGuard on the same, so let's move the daemon into its own process.